analyzers/FacsCalibur/nbproject/genfiles.properties

```diff
@@ -0,0 +1,8 @@
+build.xml.data.CRC32=eb63407c

```

Comment:
This file should not be added to version control. It belongs with the IDE (Netbeans)